### PR TITLE
Adding some basic tests that ensure that we can issue HTTP requests to the various APIs

### PR DESCRIPTION
**Pull request by christopherhafke**
These tests will also run when a `build` is done, to ensure that these don't slip by.
Perhaps these can be run by a CI tool at some point.

### CODE REVIEW

File: src/components/MailingList/MailingList.test.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+// NOTE: this test does NOT consitute a thorough test of the MailingList.
+// It simply ensures that the proper environment variables are installed.
```
Comment (eashwar) [inline src/components/MailingList/MailingList.test.jsx:4]: Agreed that this is not a thorough test. As a suggestion for truly testing the backend, perhaps a `/ping/` or `/health` endpoint which verifies the validity of the mailjet API key on the backend (and other API keys once attendance stuff is implemented)?

File: src/components/MailingList/MailingList.test.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+it("can issue a non-404-ing request to the backend", () => {
+    let mh_backend = process.env.REACT_APP_MH_BACKEND_URL;
+    let payload = {};
+    // An empty payload is a confirmed noop on the Mailjet side.
+    return axios({
```
Comment (eashwar) [inline src/components/MailingList/MailingList.test.jsx:9]: was going to ask. do we really _need_ `axios`? thoughts on refactoring to the Fetch API?

**Review by eashwar — APPROVED** [review]
LGTM for us to get started. Some questions/comments for future consideration.

File: src/components/MailingList/MailingList.test.jsx (hunk 1 above)
Comment (christopherhafke) [reply src/components/MailingList/MailingList.test.jsx:4]: Good idea, I was thinking about that when I wrote this test, that's definitely a less hacky solution.

File: src/components/MailingList/MailingList.test.jsx (hunk 2 above)
Comment (christopherhafke) [reply src/components/MailingList/MailingList.test.jsx:9]: We don't need `axios`, and I'd definitely prefer to refactor.

**Pull request merged**